go.mod

```diff
@@ -1,22 +1,22 @@
 module github.com/k8ssandra/k8ssandra-operator
 
-go 1.17
+go 1.18
 
 require (
 	github.com/Jeffail/gabs v1.4.0
 	github.com/Masterminds/semver/v3 v3.1.1
+	github.com/apache/tinkerpop/gremlin-go v0.0.0-20220530191148-29272fa563ec
 	github.com/bombsimon/logrusr/v2 v2.0.1
-	github.com/cespare/xxhash/v2 v2.1.2 // indirect
```

Comment:
I moved all indirect dependencies to the second require block below.

Reply:
While this may look prettier, sadly it's pretty futile. Next time someone runs ``go mod tidy`` , it will mess things up. Or ``go get -d``.

Reply:
Yeah, maybe we can just clean things up manually from time to time...